src/components/Select/SearchableSelect.js

```diff
@@ -59,6 +64,8 @@ export default class SearchableSelect extends React.Component {
     this.setState({
       inputValue: inputValue,
       filteredOptions: filterOptions(options, inputValue, this.props),
+    }, ()=>{
+      props.onInputValueChange(inputValue);
```

Comment:
Does this need to happen after state is updated?

Also, please add a quick existence check a la `props.onInputValueChange && props.onInputValueChange(inputValue);`

Reply:
I think so: in case the user wants to update the options, we do not want them to then be overwritten by the asynchronous setState, as unlikely as that may be.

👍 for the existence check